stdlib/public/core/AssertCommon.swift

```diff
@@ -329,6 +329,18 @@ internal func _diagnoseUnexpectedEnumCase<SwitchedValue>(
 @_semantics("unavailable_code_reached")
 @usableFromInline // COMPILER_INTRINSIC
 internal func _diagnoseUnavailableCodeReached() -> Never {
+  _diagnoseUnavailableCodeReached_aeic()
+}
+
+// FIXME: Remove this with rdar://119892482
+/// An `@_alwaysEmitIntoClient` variant of `_diagnoseUnavailableCodeReached()`.
+/// This is temporarily needed by the compiler to reference from back deployed
+/// clients.
+@_alwaysEmitIntoClient
+@inline(never)
+@_semantics("unavailable_code_reached")
+@usableFromInline // COMPILER_INTRINSIC
```

Comment:
Don't think this is needed if it's already aeic

Reply:
Yeah you're right. I'll remove it once CI has completed an initial run.